content/TrueCommand/Alerts/AlertManage.md

```diff
@@ -144,41 +143,39 @@ To properly configure SMTP email:
    
    ![AlertServicesSMTPEmailptions](/images/TrueCommand/2.0/AlertServicesSMTPEmailptions.png "Alert Services: SMTP Email options")
 
-   * **Mailserver** for example *smtp.gmail.com*
+   * **Mailserver** (*smtp.gmail.com* for example)
    * **Mailserver port** number
    * **Auth user** email address for plain authentication, for example, *adminuser@yourmail.com*
    * **Auth pass** password for the plain authentication; for a *No-Auth* SMTP configuration, leave the password field blank
-   * **From** is what sends the email (i.e., no-reply@TrueCommand.io), or allows you to customize the sender field of the email 
+   * **From** is what sends the email (i.e., no-reply@TrueCommand.io) or allows you to customize the sender field of the email 
 
   Click **Test** on the **SMTP Email** configuration screen to verify that the configuration is correct.
```

Comment:
Is this step 1 in the process? if yes, give it a 1. and move it to line 143, add a blank line, indent the image to align with the test indentation.